lib/doorkeeper/orm/active_record/application.rb

```diff
@@ -12,6 +12,14 @@ class Application < ActiveRecord::Base
     validates :uid, uniqueness: true
     validates :redirect_uri, redirect_uri: true
 
+    validates_each :scopes, if: :enforce_scopes? do |record, attr, value|
```

Comment:
Let's move it it from the `validates_each` to `validate` with custom method or just `validate` with a block (because `validates_each` is responsible for processing bunch of attributes, but we process only one - `scopes`).